include/interface/libs/VideoKit.h

```diff
@@ -15,7 +15,7 @@ class VideoKit
 
 	virtual void initializeScreen() = 0;
 
-	virtual void displayImage(const std::filesystem::path &path) = 0;
+	virtual void displayImage(const std::filesystem::path &path, bool display_white_backgroung = false) = 0;
```

Comment:
My suggestion of name: 

```suggestion
	virtual void displayImage(const std::filesystem::path &path, bool fill_background = false) = 0;
```
```suggestion
	virtual void displayImage(const std::filesystem::path &path, bool fill_with_white_background = false) = 0;
```